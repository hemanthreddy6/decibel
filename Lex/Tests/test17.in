music: audio <- load "song.wav";
loudMusic: audio <- music * 2.0;
softMusic: audio <- music * 0.5;
play loudMusic;
play softMusic;
