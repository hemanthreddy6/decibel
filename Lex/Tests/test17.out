IDENTIFIER
:
AUDIO
<-
LOAD
STRING_LITERAL
;
IDENTIFIER
:
AUDIO
<-
IDENTIFIER
*
FLOAT_LITERAL
;
IDENTIFIER
:
AUDIO
<-
IDENTIFIER
*
FLOAT_LITERAL
;
PLAY
IDENTIFIER
;
PLAY
IDENTIFIER
;
